### DISCUSSION TITLE
blog/2024-08-Neighbors-Allotment-Garden/index

### DISCUSSION THREAD

**Discussion by giscus** [basejumpa/basejumpa.github.io#7] [Show and tell]
# blog/2024-08-Neighbors-Allotment-Garden/index

https://basejumpa.github.io/blog/2024-08-Neighbors-Allotment-Garden/index.html

**Comment by CasimirBo** [comment]
Hi,
mabey this is an interesting repo for this task:
https://github.com/rjsf-team/react-jsonschema-form

**Reply by basejumpa** [reply]
@CasimirBo Thx for the precious hint. I like the declarative approach to define the form. I definitely will integrate it!
